TundraTD/Assets/Scripts/Camera/CameraMovement.cs

```diff
@@ -0,0 +1,97 @@
+﻿using UnityEngine;
+
+namespace Camera
+{
+    public class CameraMovement : MonoBehaviour
+    {
+        
+        [SerializeField] private float minimalInertiaThreshold = .125f;
+        [SerializeField] private float maximalCameraMoveThreshold;
+        [SerializeField] private float inertiaMultiplier;
+        [SerializeField] private float minimalCameraSize = 1;
+        [SerializeField] private float maximumCameraSize = 8;
+
+        private Vector3 _inertiaDirection;
+        private Vector3 _cameraSpeed;
+        private Vector3 _touchStart;
+        private UnityEngine.Camera _mainCamera;
```

Comment:
You could replace UnityEngine.Camera with just a Camera
```suggestion
        private Camera _mainCamera;
```